client/dist/style.css

```diff
@@ -5,12 +5,16 @@
   align-items: center
 }
 
-.carousel-wrapper {
+.image-wrapper {
```

Comment:
Maybe you already have this somewhere else, but you can also implement "scroll-behavior: smooth" for a smoother scroll (if that's what the Yelp page does)